addons/base_import/models/base_import.py

```diff
@@ -987,6 +1006,7 @@ def parse_preview(self, options, count=10):
             return {
                 'fields': fields_tree,
                 'matches': matches or False,
+                'lang_warn': lang_warn or False,
```

Comment:
Note that it is not really a lang_warn... and we could always return lang information (even if guess_lang == context lang), like
```
'lang_information': {
    'lang_code': code,
    'lang_name': name,
    'lang_is_different': lang_code != self.env.context['lang'],
}
```
And template would simply check for lang_is_different. That way we always have this information, leading to a less different set of rendering values depending on usage.

Reply:
Changed and updated the test. That way, it could indeed be used for other purpose.